### PR TITLE
Feature

### PR DESCRIPTION
**Pull request by atenaiis**
Good afternoon,

The goal of this project is to demonstrate mastery with DOM manipulation by dynamically rendering a simple restaurant's homepage.

Please follow read me file instructions to check this project.

Cheers

Mariana Campos

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,51 @@
...
+- CSS3
+
+
+## Getting Started
+
+To get a local copy of this project running follow these simple steps.
+
+- Clone this repository
+ > `git clone < git@github.com:atenaiis/restaurant-pg.git >
+- Navigate to the Repo Folder
```
Comment (afanimartin) [inline README.md]: - Add command for doing that

File: README.md [hunk 2]
```diff
@@ -1 +1,51 @@
...
+- To set Webpack and the dependencies run the command
+> `npm install`
+- Use your favorite browser to open the index.html file (located at the dist page)
```
Comment (afanimartin) [inline README.md]: - dist page or folder? Be more explicit in your instructions.

File: README.md [hunk 3]
```diff
@@ -1 +1,51 @@
...
+
+
+## 🤝 Contributing
+
+Contributions, issues and feature requests are welcome!
+
+Feel free to check the isuue.
+
+
+## Show your support
```
Comment (afanimartin) [inline README.md:53]: - Add open-source license such as MIT

File: src/contact.js [hunk 4]
```diff
@@ -0,0 +1,28 @@
+import cafe from './images/postre2.jpg';
+
+const contactText = '<p> Oaxaca, Mexico<br>Great place to live\n<br><strong>Phone:</strong> 951 - 647 - 7957<br><strong>Fax:</strong> 951 - 647 - 7957<br><strong>Email:</strong> getintocuh@camarellas.com<br></p>';
+
+const contactSection = () => {
+  const divContact = document.createElement('div');
+
+  // Set H1
```
Comment (afanimartin) [inline src/contact.js:8]: - Remove all comments here

File: src/home.js [hunk 5]
```diff
@@ -0,0 +1,27 @@
+import family from './images/postre1.jpg';
+
+const homeSection = () => {
+  const homeText = "Camarella's is a place where you will find home made products using fresh and local products";
+  const homesection = document.createElement('div');
+
+  // Set H1
```
Comment (afanimartin) [inline src/home.js]: - Remove comments

File: src/menu.js [hunk 6]
```diff
@@ -0,0 +1,28 @@
+const menuSection = () => {
+  // Set new Div
+  const divMenu = document.createElement('div');
+
+  // Set H1
```
Comment (afanimartin) [inline src/menu.js]: - Remove comments

File: src/load.js [hunk 7]
```diff
@@ -0,0 +1,38 @@
+import Icon from './images/organicos2.jpg';
+import homeSection from './home';
+
+const myBody = document.getElementById('my-body');
+const divContent = document.getElementById('content');
+
+// Creating Website Logo
```
Comment (afanimartin) [inline src/load.js]: - Remove comments

File: src/index.js [hunk 8]
```diff
@@ -0,0 +1,33 @@
+// import './css/style.css';
+import load from './load';
+import homeSection from './home';
+import menuSection from './menu';
+import contactSection from './contact';
+
+load();
+
+// Getting Div Sections where the elements will be appended
```
Comment (afanimartin) [inline src/index.js]: - Remove all comments

**Review by afanimartin — CHANGES_REQUESTED** [review]
### Changes Required 🙏🏿 🙏🏿 🙏🏿 
Nice work on this PR. 
### Highlights 🌵 🌵 🌵 
- Use of modules
- Required packages added to `package.json`

### Changes 🙅🏿 🙅🏿 🙅🏿 
- Remove all comments from the codebase. They make your code look ugly
- Update the readme. Check comments


### Martin Afani
#### Slack: whiz25

**Review by rukundoeric — APPROVED** [review]
# Status: 🍾 PROJECT APPROVED 🍾
_Excellent job on this project guys. 🎆 Keep it up._

- ### To Highlight
  - Follows gitwork-flow. 💯 
  - App is functionsl and available online. 💯 
  - It looks nice 💯
  
- ### To Improve
  - N/A

_Congratulations on the splendid job **@team** 👍
You can now merge this PR and submit the project completion form 🚀 . I wish you all the very best in your next tasks 🥇_

### _Keep working hard._
### _Happy Coding😃_
-----------------------------------------------------------------------------------------------------------------------------------------
**Regards,
[Rukundo Eric](https://rukundo-eric.herokuapp.com/) ,
[LinkedIn](https://www.linkedin.com/in/rukundo-eric/),
Slack: Rukundo Eric**

**Pull request merged**